licenses/teamspeak3

```diff
@@ -1,443 +1,1141 @@
-TeamSpeak 3.x - End User License Agreement
```

Comment:
I'm confused why you're changing this file. Is this a license update or sth? I don't think we tend to edit licenses in place. Also, commit message summary line is too long.

Reply:
Yes, it's an license update. Because the license, which is currently in portage, is pretty old. It's not even for the current version in portage valid. So it should be anyway updated. This license is shipped  for the current release in portage and for the new release 3.1.1.

Reply:
Ok, that addresses my concerns. Could you then make the commit message indicate it's an update? Also stating that the current license is not valid for current version would be helpful.

Reply:
Will do.